### PR TITLE
Manfred/polish interaction code

### PR DESCRIPTION
**Pull request by mj3cheun**
- clean up white space (remove all mixed tabs and spaces and standardize to 4 space tab)
- component returns removeEventListener function

### CODE REVIEW

**Comment by shammamah-zz** [comment]
@mj3cheun Please include a brief bullet-point summary of the changes you made in the initial PR comment!

File: src/interactions.js [hunk 1]
```diff
@@ -1,87 +1,80 @@
 "use strict";
 
-var speckView = require('./view.js'); 
+var speckView = require("./view.js");
 
 module.exports = function(component, renderer, container) {
     component.setState({
-	interactions: {
```
Comment (shammamah-zz) [inline src/interactions.js:7]: Why is this diff here?

Comment (mj3cheun) [reply src/interactions.js:7]: this is just some whitespace cleanup

**Review by shammamah-zz — APPROVED** [review]
:dancer: great work :)

**Pull request merged**